frontend/controller/actions/group-kv.js

```diff
@@ -46,21 +60,22 @@ export default (sbp('sbp/selectors/register', {
       throw new Error('Unable to update lastLoggedIn without an active session')
     }
 
+    const now = sbp('chelonia/time') * 1000
```

Comment:
... what is the timescale returned by '`chelonia/time'`? Isn't it the same as `Date.now()`?

```
> new Date(Date.now())
2025-02-07T19:48:52.406Z
> new Date(Date.now()*1000)
+057075-05-18T15:42:05.000Z
> 
```

Reply:
Alright, we can return to this later maybe, opened discussion for it: #2593